### PR TITLE
bulk delete of tpr packages

### PR DESCRIPTION
**Pull request by deads2k**
related to https://github.com/kubernetes/kubernetes/issues/48152

Bulk delete of the TPR code.  I made the minimal changes outside the delete to try to keep it easy to review.

### CODE REVIEW

File: pkg/master/master.go [hunk 1]
```diff
@@ -259,7 +258,7 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget,
 		batchrest.RESTStorageProvider{},
 		certificatesrest.RESTStorageProvider{},
 		// TODO(enisoc): Remove crdRESTOptionsGetter input argument when TPR code is removed.
```
Comment (enisoc) [inline pkg/master/master.go]: Are you going to remove `crdRESTOptionsGetter` in a later PR?

**Review by enisoc — APPROVED** [review]
/lgtm

File: pkg/master/master.go (hunk 1 above)
Comment (deads2k) [inline pkg/master/master.go]: > Are you going to remove crdRESTOptionsGetter in a later PR?

Yes.  I think the rest of changes ripple out to code we're keeping, so they'll need more careful review.

**Pull request merged**